pywt/_wavelet_packets.py

```diff
@@ -68,6 +68,12 @@ def __init__(self, parent, data, node_name):
 
         # data - signal on level 0, coeffs on higher levels
         self.data = data
+        # Need to retain original data size/shape so we can trim any excess
+        # boundary coefficients from the inverse transform.
+        if self.data is None:
+            self.data_shape = None
```

Comment:
Should this be `_data_shape`? No reason for this to be a public attribute I think.